### PR TITLE
map api implementation using leaflet.js library and maptiler api

### PR DESCRIPTION
**Pull request by KenStephens**
### Screenshots
image
)

### Changes
-Implemented Leaflet js Library - opensource
-Implemented Maptiler API for map-
-Created folder housing map component and osm-provider assets
-Added Map container CSS to App.css file

#### How to test
- To test at the BaseMap component to the App.jsx routes with path "/map", import BaseMap, then navigate to "/map" where map should render.

### CODE REVIEW

**Review by RL1995 — APPROVED** [review]
All changes look up to standard

File: src/index.css [hunk 1]
```diff
@@ -66,3 +66,9 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+/* mapcontainer */
```
Comment (jmedina1329) [inline src/index.css:71]: Does this map container need to be in index? Or should you create a css module for your map?

I'm asking, "Is this container css something that will be used by other devs and therefore should be open to all via index.css or is this specific to your work, and should be in it's own file?"

Comment (KenStephens) [reply src/index.css:71]: Yes, it can I'll update the code.

**Review by RL1995 — COMMENTED** [review]
Required changes made to CSS portion.

File: src/index.css (hunk 1 above)
Comment (KenStephens) [reply src/index.css:71]: REMOVED from index.css

**Review by jmedina1329 — APPROVED** [review]

**Pull request merged**